frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -81,34 +81,39 @@ async function test_previously_created_drafts_rendered(page: Page): Promise<void
         await common.get_text_from_selector(
             page,
             ".draft-row .message_header_stream .stream_label",
+            true,
```

Comment:
Would it be convenient to have a calling convention here that included the `only_visible` parameter name here? The bare `true`, without context, is hard to read without looking up the function itself. The other thing that could help is having the calling convention not require any changes if one wants the default of ignoring hidden elements.

Relatedly, maybe we want to rename `only_visible=true` as the default to `include_hidden=false`?